python/simpler/worker.py

```
@@ -1139,6 +1320,11 @@ def _run_chip_main_loop(  # noqa: PLR0912, PLR0913, PLR0915 -- unified TASK_READ
                 control_shm.close()
             except Exception:  # noqa: BLE001
                 pass
+        for host_shm, _lo, _hi, _base in host_buf_table.values():
+            try:
+                host_shm.close()
+            except Exception:  # noqa: BLE001
+                pass
```

Ruff (0.15.20)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)